Automattic/jetpack
# Jetpack Search: Add new marketplace related fields to postmeta_to_sync array

Related to https://github.com/Automattic/wp-calypso/issues/69948





#### Changes proposed in this Pull Request:

* Adds two new meta fields to the sync allow list

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






- If possible, set up a local Jetpack environment with this branch applied
- Add a post with custom fields with the name `wpcom_marketplace_store_product_monthly_id` and `wpcom_marketplace_store_product_yearly_id` 
- Check that the field has been synchronized to WP.com
